test/registry/install-plugin/installPlugin.t.sol

```diff
@@ -9,9 +9,8 @@ import { Registry_Test } from "../Registry.t.sol";
 contract InstallPlugin_Test is Registry_Test {
     function test_RevertWhen_CallerDoesNotHaveProxy() external {
         vm.expectRevert(
-            abi.encodeWithSelector(IPRBProxyRegistry.PRBProxyRegistry_CallerDoesNotHaveProxy.selector, users.bob)
+            abi.encodeWithSelector(IPRBProxyRegistry.PRBProxyRegistry_CallerDoesNotHaveProxy.selector, users.alice)
         );
```

Comment:
Alice is the default user so I'd prefer to use Bob in this test to minimize the chance of this test failing in the future

Reply:
Not sure about what "future" you're talking, but if you end up going for Alice, note that "changing prank" after the call to `vm.expectRevert()` is being ignored by Foundry.

Reply:
In other words, before my changes, Alice was still the `msg.sender` in this tests.

Which begs a question... How has this test ever been successful?.. 🤔
Alice must've been the `msg.sender` - so, how has the `revert` with `users.bob` encoded into the error log been accespted by Foundry?..

Reply:
Foundry does not ignore the `changePrank` - see these:

- https://ethereum.stackexchange.com/q/147319/24693
- https://github.com/foundry-rs/foundry/issues/4875

Reply:
> Not sure about what "future" you're talking

It's about the _potential_ for making errors - which is always present.

Using Bob means relying less on the background environment to be configured properly.

Reply:
I haven't found anything in the references above to adress the topic I've opened. That is, calling `vm.changePrank()` after `vm.expectRevert()`.

See the "After calling expectRevert, calls to other cheatcodes before the reverting call are ignored." bit in the [documentation on `vm.expectRevert()`](https://book.getfoundry.sh/cheatcodes/expect-revert?highlight=expectRevert#expectrevert)

Reply:
Shoot - you are absolutely right. Good catch, and thanks for correcting my error(s).

[I've posted this question in the Foundry Support group](https://t.me/foundry_support/40162) to triple-check our understanding, but based on the docs you linked, you are very likely correct.

Reply:
removed the `changePrank` and modified the test to use Alice, as you have originally done in the PR

Reply:
Awesome!

Reply:
It turns out that this was more complicated than we thought:

- https://t.me/foundry_support/40206
- https://github.com/foundry-rs/book/issues/948

Reply:
Wicked!